conformance/.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [main]
+  merge_group:
+
+env:
+  HICKORY_REV: 107635c6c5934524894736f1b141198d0fa62fec
```

Comment:
any reason to pin this to a version like this?

Reply:
It is done to avoid bothering contributors with fixing broken tests they are unfamiliar with as this conformance test suite is a new addition.

Reply:
> any reason to pin this to a version like this?

this whole directory should have been removed but I forgot to do so. it contains the CI settings used in the dnssec-tests repo. we used a pinned version of hickory-dns because we didn't have its source as part of the repository. this workflow file is not read by the GHA worker.

the new CI workflow, in `/.github/workflows/conformance.yml`, uses the latest version of hickory-dns which is what's included in the checkout of the repository.